qldbdriver/result.go

```diff
@@ -19,8 +19,17 @@ import (
 	"github.com/aws/aws-sdk-go/service/qldbsession"
 )
 
-// Result is a cursor over a result set from a QLDB statement.
-type Result struct {
+// Result is an interface for a Result Set cursor.
```

Comment:
This should keep the old, existing docstring.

Reply:
The `Result struct` got renamed to `result struct` which has the same docstring.
This new docstring is for the `Result interface`. Are you saying this interface should have the same docstring as the struct?

Reply:
Addressed.